### PR TITLE
[Binding addons starting with "P"] Correct linter errors only

### PR DESCRIPTION
**Pull request by bgilmer77**
Signed-off-by: Brad Gilmer <brad@gilmer.tv>

### CODE REVIEW

File: addons/binding/org.openhab.binding.pulseaudio/README.md [hunk 1]
```diff
@@ -6,38 +6,34 @@ This binding integrates pulseaudio devices.
...
 
-| Channel Type ID | Item Type    | Description  |
-|-----------------|------------------------|--------------|----------------- |------------- |
-| volume | Dimmer  | Volume of an audio device in percent |
-| mute | Switch | Mutes the device |
-| state | String | Current state of the device (suspended, idle, running, corked, drained) |
-| slaves | String | Slave sinks of a combined sink |
-| routeToSink | String | Shows the sink a sink-input is currently routed to |
+| Channel Type ID | Item Type | Description                                                             |   |   |
+|-----------------|-----------|-------------------------------------------------------------------------|---|---|
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.pulseaudio/README.md]: Plesse remove the empathie columns

File: addons/binding/org.openhab.binding.pulseaudio/README.md [hunk 2]
```diff
@@ -6,38 +6,34 @@ This binding integrates pulseaudio devices.
...
+| volume          | Dimmer    | Volume of an audio device in percent                                    |   |   |
+| mute            | Switch    | Mutes the device                                                        |   |   |
+| state           | String    | Current state of the device (suspended, idle, running, corked, drained) |   |   |
+| slaves          | String    | Slave sinks of a combined sink                                          |   |   |
+| routeToSink     | String    | Shows the sink a sink-input is currently routed to                      |   |   |
 
+<!--ToDO
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.pulseaudio/README.md]: What should we do with this?

Comment (bgilmer77) [reply addons/binding/org.openhab.binding.pulseaudio/README.md]: For now, let's just leave it alone.  In the original version it just said, ** Example, and that was the end of the document.  So obviously whoever wrote the binding intended to put an example, but it was never completed.  Seemed like the right thing to do was to note that this is unfinished.  I am only concentrating on linter errors and hope to get through all the bindings before 2.2 is released, so I would like to keep focused on that for now.

File: addons/binding/org.openhab.binding.pulseaudio/README.md (hunk 1 above)
Comment (bgilmer77) [reply addons/binding/org.openhab.binding.pulseaudio/README.md]: This will be fixed in my next commit.

**Pull request merged**